executor/common_fuchsia.h

```diff
@@ -185,12 +185,14 @@ long syz_mmap(size_t addr, size_t size)
 	status = zx_vmo_replace_as_executable(vmo, ZX_HANDLE_INVALID, &vmo);
 	if (status != ZX_OK) {
 		debug("zx_vmo_replace_as_executable failed with: %d\n", status);
+		zx_handle_close(vmo);
```

Comment:
This is unnecessary:

> handle is always invalidated.

--https://fuchsia.dev/fuchsia-src/zircon/syscalls/vmo_replace_as_executable

Reply:
Thanks for pointing this out -- I didn't know!